backend/src/lastwill/lastwill.service.spec.ts

```diff
@@ -0,0 +1,122 @@
+import { LastWill, PersonType } from '../db/entities/lastwill.entity'
+import { LastWillService } from './lastwill.service'
+
+const SAMPLE_LAST_WILL: LastWill = {
+  _id: 'aaaaaaaaaaaaaaaa',
+  accountId: 'aaaaaaaaaaaaaab',
+  common: {
+    isBerlinWill: false,
+    isPartnerGermanCitizenship: true,
+  },
+  testator: {
+    name: 'Peter Lustig',
+    address: {
+      city: 'here',
+    },
+  },
+  items: [
+    {
+      id: 'a',
+      name: 'my old bike',
+    },
+  ],
+  financialAssets: [
+    {
+      id: 'b',
+      amount: 10000,
+      where: 'over there',
+      currency: 'imagination',
+    },
+  ],
+  heirs: [
+    {
+      id: 'person1',
+      type: PersonType.FATHER,
+      name: 'Anders Lustig',
+      percentage: 100,
+      itemIds: ['a'],
+    },
+  ],
+  progressKeys: [],
```

Comment:
We don't necessarily have to do that. The validation is done in the frontend and a crafted request leads to a las will with placeholders. There is no reason to prevent that